lib/python/qmk/search.py

```diff
@@ -61,10 +61,22 @@ def apply(self, target_info: TargetInfo) -> bool:
 
 class Length(FilterFunction):
     func_name = "length"
+    funcMap = [
+         (">=", int.__ge__),
+         ("<=", int.__le__),
+         (">", int.__gt__),
+         ("<", int.__lt__),
+     ]
 
     def apply(self, target_info: TargetInfo) -> bool:
         _kb, _km, info = target_info
-        return (self.key in info and len(info[self.key]) == int(self.value))
+        val = int(''.join(c for c in self.value if c.isdigit()))
```

Comment:
As far as i can tell this is intended to convert the (potentially) `{<,<=,>,>=}number` into an actual `int`, but i dont like it because of two things:
- You will mistakenly convert weird inputs into numbers, eg: `3a2` will be parsed as `32`
- Is much less readable than just doing
```py
for start, func in self.funcMap: # i would also name this something like "operator"
    if self.value.startswith(start):
        comp = func
        val = int(self.value[len(func):]  # remove prefix, then convert to int (or fail trying to do so)
```
Also, the extra parenthesis on the for, if and return look a bit off, but just code style yet again